File: web3rpc/api/code-samples/python/klay/others/createAccessList.py
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

callObject = {
      "from": "0x3bc5885c2941c5cda454bdb4a8c88aa7f248e312",
      "to": "0x00f5f5f3a25f142fafd0af24a754fafa340f32c7",
      "gas": "0x3d0900",
      "gasPrice": "0x3b9aca00",
      "data": "0x20965255"
    }
blockTag = "latest"

sdk = OpenSDK(host)
klay_response = sdk.klay.create_access_list(callObject, blockTag)

print(klay_response)

File: web3rpc/rpc-specs/code-samples/javascript/klay/others/createAccessList.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");

    const callObject = {
        "from": "0x3bc5885c2941c5cda454bdb4a8c88aa7f248e312",
        "to": "0x00f5f5f3a25f142fafd0af24a754fafa340f32c7",
        "gas": "0x3d0900",
        "gasPrice": "0x3b9aca00",
        "data": "0x20965255"
    }
    const blockParameter = 'latest'
    provider.klay.createAccessList(callObject, { blockParameter }, (err, data, response) => {
        console.log(data);
    });
}
)()